mtags/src/main/scala-3/scala/scala/meta/internal/metals/ClassFinder.scala

```diff
@@ -14,9 +16,11 @@ import dotty.tools.dotc.core.Flags
 
 object ClassFinder {
 
-  class SymbolTraverser(offset: Int) extends UntypedTreeTraverser {
+  class SymbolTraverser(offset: Int, fileName: String)
+      extends UntypedTreeTraverser {
     var symbol = ""
     var isInnerClass: Boolean = false
+    var isOuter: Boolean = true
```

Comment:
What does it mean to "be outer"?

Reply:
Might be better to have `toplevel` - I will change that.

Reply:
Fixed!